site/src/pages/LoginPage/LoginPageView.tsx

```diff
@@ -28,13 +29,28 @@ export const LoginPageView: FC<LoginPageViewProps> = ({
   // This allows messages to be displayed at the top of the sign in form.
   // Helpful for any redirects that want to inform the user of something.
   const info = new URLSearchParams(location.search).get("info") || undefined;
+  const logoURL = getLogoURL();
+  const applicationLogo = logoURL ? (
+    <div>
+      <img
+        alt=""
```

Comment:
For accessibility reasons, could the alt text could be set to the return value of `getApplicationName`?

Reply:
Done